src/plugin/localeData/index.js

```diff
@@ -23,7 +29,8 @@ export default (o, c, dayjs) => { // locale needed later
         (instance ? instance.format('dd') : getShort(this, 'weekdaysMin', 'weekdays', 2)),
       weekdaysShort: instance =>
         (instance ? instance.format('ddd') : getShort(this, 'weekdaysShort', 'weekdays', 3)),
-      longDateFormat: format => this.$locale().formats[format]
+      longDateFormat: format => getLongDateFormat(format)
```

Comment:
we should pass instance locale, right?  this.$locale()

Reply:
I added those to the function.